### PR TITLE
Update lra 1 2 0

### PR DESCRIPTION
**Pull request by mchaisso**
Update of lra to be consistent with our resubmitted manuscript.
----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by fanninpm** [comment]
@dpryan79 This recipe seems to be incompatible with BiocondaBot's update mechanism. Please advise.

**Comment by mchaisso** [comment]
@Jingwen7 Adding to the list

**Comment by fanninpm** [comment]
@BiocondaBot please add label

**Comment by fanninpm** [comment]
@BiocondaBot please update

File: recipes/lra/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   name: lra
   version: {{ version }}
 
 source:
   url: https://github.com/ChaissonLab/LRA/archive/V{{ version }}.tar.gz
-  sha256: 0ec863e659e42509edcf77e3b3f2bd8f1fed5e4d7ada2c1be44ecc83c2b0ea05
+  sha256: de2bdc18b0886ee7a934c4daf6c18d6dd796e843b31e68923c948ea02c2828d7
 
 build:
   number: 1
```
Comment (fanninpm) [inline recipes/lra/meta.yaml:12]:
```suggestion
  number: 0
```

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
Please disable circleci on your fork, our CI testing cannot currently run.

**Comment by fanninpm** [comment]
@mchaisso go to https://github.com/ChiassonLab/bioconda-recipes/settings/actions and click the radio button next to "Disable Actions".

**Comment by mchaisso** [comment]
Done, thank you for finding this!

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by fanninpm** [comment]
> `ERROR: Permission to ChaissonLab/bioconda-recipes.git denied to dpryan79.`

Does BiocondaBot have permission to push commits to your repo?

**Comment by mchaisso** [comment]
I just added BiocondaBot to the repo with write permissions.

**Comment by fanninpm** [comment]
I think you also have to be the one that calls the bot.

**Comment by mchaisso** [comment]
@BiocondaBot update

**Comment by fanninpm** [comment]
I may be wrong, but I don't think it works without the magic word.

**Comment by mchaisso** [comment]
@BiocondaBot please update

**Comment by mchaisso** [comment]
That's hilarious. I was referencing an older thread:
https://github.com/bioconda/bioconda-recipes/issues/14400

**Comment by fanninpm** [comment]
https://bioconda.github.io/contributor/faqs.html#circleci-macos-plans

**Comment by mchaisso** [comment]
Is there a way to get the BiocondaBot to accept a pending invitation?

**Comment by dpryan79** [comment]
> Is there a way to get the BiocondaBot to accept a pending invitation?

Not easily, I might have the login information somewhere maybe.

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by fanninpm** [comment]
@BiocondaBot please update

**Comment by mchaisso** [comment]
I just sent the invite; Devon hasn't accepted yet.

**Comment by dpryan79** [comment]
@bioconda-bot please update (maybe this will work?)

**Comment by fanninpm** [comment]
Superseded by #28444

**Pull request closed without merging**